### PR TITLE
[iOS] Add the contributor view

### LINKED ISSUE #493: iOS App contributor view
## Kind (Required)
- Proposal / Discussion

 ## Overview (Required)
- A view that shows contributors 

 ## Links
- [Design](https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=22%3A402)

### PR DESCRIPTION
**Pull request by woxtu**
## Issue
- Close #493 

## Overview (Required)
- Same the title
- Sticky section header has not been implemented

## Links
- [Design](https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=22%3A402)

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by ry-itto** [comment]
@woxtu 
Before reviewing, please merge master branch 🙏 
Master branch has disruptive change about `ios_combined` framework.
That framework is renamed to `ioscombined`.

**Comment by woxtu** [comment]
@ry-itto Ready!

File: ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift [hunk 1]
```diff
@@ -119,7 +119,11 @@ final class SidebarViewController: UITableViewController {
             let vc = SponsorViewController()
             transition(to: vc)
         case .contributor:
-            break
+            if rootViewController.viewControllers.first is ContributorViewController {
+                break
+            }
+            let contributorViewController = ContributorViewController.instantiate()
+            rootViewController.pushViewController(contributorViewController, animated: true)
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift]: In my opinion, the view moved from sidebar should have `menuButton` and not to move with navigation.
And views moved from sidebar's transition way should be same I think.

So, can you move view with `transition(to:)` function? 🙏

File: ios-base/DroidKaigi 2020/Contributor/Views/ContributorViewDataSource.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            preconditionFailure()
+        }
+
+        let contributor = items[indexPath.section].contributors[indexPath.item]
+
+        if let url = URL(string: contributor.iconUrl) {
+            let options = ImageLoadingOptions(transition: .fadeIn(duration: 0.3))
+            Nuke.loadImage(with: url, options: options, into: cell.iconImageView)
+        }
+        cell.iconImageView.layer.cornerRadius = cell.iconImageView.frame.height / 2
```
Comment (ry-itto) [inline ios-base/DroidKaigi 2020/Contributor/Views/ContributorViewDataSource.swift]: This is nits.
This code can write in `iconImageView`'s `didSet` in `ContributorCell ` 😄

File: ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift (hunk 1 above)
Comment (woxtu) [reply ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift]: Sorry, I didn't see other branches. I'll fix like that.

File: ios-base/DroidKaigi 2020/Contributor/Views/ContributorViewDataSource.swift (hunk 2 above)
Comment (woxtu) [reply ios-base/DroidKaigi 2020/Contributor/Views/ContributorViewDataSource.swift]: Right.

**Pull request merged**